### ISSUE TITLE
[med] Color Contrast: Chat: Grey icons on grey background fail color contrast requirements

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5651]
## Action Performed:
Prerequisites:
- User has created personal account and is logged into same.
- User has multiple chats in chat history. Some pinned, some with message entered but not sent, at least 1 with a group of maximum participants (8).

1. Using Chrome, open https://staging.new.expensify.com/
2. Note the search icon button, status icons for pinned chats (pin), chats with draft messages (pencil) and group participant badge indicator numbers, (+5). Measure the contrast of the dark grey icons against the adjacent light grey background and the reverse color pattern for the group badge number (+5).

## Expected Result:
The graphical information required to identify user interface components must have a contrast ratio of at least 3.0:1 against adjacent colors. The dark grey icons must have contrast ratio of at least 3.0:1 against the adjacent light grey background. The same is true for the light grey text (+5) against the dark grey color of the circular icon.

## Actual Result:
The grey icons, the grey circle for the participant badge (+5) (#C6C9CA) against the light grey background and the participant number (+5) (#FAFAFA) have a contrast ratio of 1.59:1.
When the chat is currently selected, the grey pencil and pin icons (#C6C9CA) against a dark grey background (#ECECEC) have a color contrast ratio of 1.40:1.

## Workaround:
No workaround

## Area issue was found in:
Chat page

## Failed WCAG checkpoints
1.4.11

## User impact:
Many color blind and sight impaired users will not be able to see the almost invisible (1.0:1) non-text indicators.

## Suggested resolution:
Use a darker shade of grey for the icons and badge circle like to bring the ratio above 3.0:1. One example of color to use is #7E8A8B which will give a color contrast ratio of 3.41:1 against #FAFAFA and 3.01:1 against #ECECEC.
https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**
![Bug5185396_Color_Contrast-Chat-grey_status_icons_against_light_grey_fails_contrast_ratio_upd](https://user-images.githubusercontent.com/88733897/136043703-c14a61bc-89c3-422c-a7a5-6f90f5fb4c45.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
